BrowserServicesKit/Sources/NetworkProtection/Routing/VPNRoutingRange.swift

```diff
@@ -46,38 +46,51 @@ public enum VPNRoutingRange {
     ]
 
     public static let publicNetworkRange: [NetworkProtection.IPAddressRange] = [
-        "1.0.0.0/8",
-        "2.0.0.0/8",
-        "3.0.0.0/8",
-        "4.0.0.0/6",
-        "8.0.0.0/7",
-        "11.0.0.0/8",
-        "12.0.0.0/6",
-        "16.0.0.0/4",
-        "32.0.0.0/3",
-        "64.0.0.0/2",
-        "128.0.0.0/3",
-        "160.0.0.0/5",
-        "168.0.0.0/6",
-        "172.0.0.0/12",
-        "172.32.0.0/11",
-        "172.64.0.0/10",
-        "172.128.0.0/9",
-        "173.0.0.0/8",
-        "174.0.0.0/7",
-        "176.0.0.0/4",
-        "192.0.0.0/9",
-        "192.128.0.0/11",
-        "192.160.0.0/13",
-        "192.169.0.0/16",
-        "192.170.0.0/15",
-        "192.172.0.0/14",
-        "192.176.0.0/12",
-        "192.192.0.0/10",
-        "193.0.0.0/8",
-        "194.0.0.0/7",
-        "196.0.0.0/6",
-        "200.0.0.0/5",
-        "208.0.0.0/4",
+        "1.0.0.0/8",        /* 1.0.0.0 - 1.255.255.255 */
```

Comment:
Diff sucks as it is, but I added these comments to make the ranges easier to track visually.

The only change should be that `168.0.0.0/6` is replaced by subranges not including link-local.  One thought is an AI make make it easier to review before vs affter.

Reply:
Also we should really have tests for these, so if you feel strongly now is the time to add them let me know and i'll do that.